app/src/main/java/org/mozilla/fenix/settings/search/EditCustomSearchEngineFragment.kt

```diff
@@ -0,0 +1,180 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.settings.search
+
+import android.net.Uri
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.Menu
+import android.view.MenuInflater
+import android.view.MenuItem
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.lifecycleScope
+import androidx.navigation.fragment.findNavController
+import kotlinx.android.synthetic.main.custom_search_engine.*
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import mozilla.components.browser.search.SearchEngine
+import org.mozilla.fenix.R
+import org.mozilla.fenix.components.FenixSnackbar
+import org.mozilla.fenix.components.searchengine.CustomSearchEngineStore
+import org.mozilla.fenix.ext.increaseTapArea
+import org.mozilla.fenix.ext.requireComponents
+import org.mozilla.fenix.settings.SupportUtils
+import java.util.Locale
+
+class EditCustomSearchEngineFragment : Fragment() {
```

Comment:
We can use the new fragment constructor and remove `onCreateView`

```suggestion
class EditCustomSearchEngineFragment : Fragment(R.layout.fragment_add_search_engine) {
```